### PR TITLE
FE-1362: Update code scan

### PR DESCRIPTION
**Pull request by kchu1337**
Note: Give your PR a short title. Something like "_TICKET NUMBER: feature description_" is good.

### What Changed
- Updated configs to match confluence documentation: https://sparkpost.atlassian.net/wiki/spaces/ENG/pages/688586768/GitHub+Security+Scanning+CodeQL+for+Javascript

### Results
https://github.com/SparkPost/2web2ui/security/code-scanning?query=tool%3ACodeQL+ref%3Arefs%2Fpull%2F2054%2Fmerge

### CODE REVIEW

**Comment by bkemper** [comment]
@kchu1337 Looks like there are "2 analyses not found".  Is this ready for review or should that be fixed first?

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -6,28 +6,31 @@ on:
...
         uses: actions/checkout@v2
-        with:
-          # We must fetch at least the immediate parents so that if this is
-          # a pull request then we can checkout the head.
-          fetch-depth: 2
-
-      # If this run was triggered by a pull request event, then checkout
-      # the head of the pull request instead of the merge commit.
-      - run: git checkout HEAD^2
-        if: ${{ github.event_name == 'pull_request' }}
```
Comment (bkemper) [inline .github/workflows/codeql-analysis.yml:26]: What is the reasoning for removing this?

Comment (kchu1337) [reply .github/workflows/codeql-analysis.yml:26]: I just followed the confluence doc. Was going to let it run and see if it works. If not, then I would add it back in. 
Also, I wasn't 100% sure what this does. If you squash all the commits into the merge commit, isn't it the same thing?

**Review by bkemper — APPROVED** [review]
LGTM, with the assumption that we won't have a successful run until we merge to master and have something to diff against.

**Pull request merged**